generators/client/templates/angular/src/main/webapp/app/layouts/profiles/_profile.service.ts

```diff
@@ -29,19 +28,23 @@ import { ProfileInfo } from './profile-info.model';
 export class ProfileService {
 
     private profileInfoUrl = <% if (authenticationType === 'uaa') { %>'<% } else { %>SERVER_API_URL + '<% } %>api/profile-info';
+    private profileInfo: Promise<ProfileInfo>;
```

Comment:
This can be done with Observable as well and I personally would prefer that to keep things consistent